includes/admin/admin-experiments.php

```diff
@@ -0,0 +1,328 @@
+	 */
+	class SCF_Admin_Experiments {
+
+		/**
+		 * Contains an array of admin experiment instances.
+		 *
+		 * @var array
+		 */
+		private $experiments = array();
+
+		/**
+		 * The active experiment.
+		 *
+		 * @var string
+		 */
+		private $active = '';
+
+		/**
+		 * This function will setup the class functionality
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function __construct() {
+			// Temporarily disabled - will be enabled when experiments feature is ready
+			// add_action( 'admin_menu', array( $this, 'admin_menu' ), 20 );
+		}
+
+		/**
+		 * This function will store an experiment class instance in the experiments array.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @param   string $experiment Class name.
+		 * @return  void
+		 */
+		public function register_experiment( $experiment ) {
+			$instance                             = new $experiment();
+			$this->experiments[ $instance->name ] = $instance;
+		}
+
+		/**
+		 * This function will return an experiment class or null if not found.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @param   string $name Name of experiment.
+		 * @return  mixed (SCF_Admin_Experiment|null)
+		 */
+		public function get_experiment( $name ) {
+			return isset( $this->experiments[ $name ] ) ? $this->experiments[ $name ] : null;
+		}
+
+		/**
+		 * This function will return an array of all experiment instances.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  array
+		 */
+		public function get_experiments() {
+			return $this->experiments;
+		}
+
+		/**
+		 * This function will add the SCF experiments menu item to the WP admin
+		 *
+		 * @type    action (admin_menu)
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function admin_menu() {
+			// bail early if no show_admin
+			if ( ! acf_get_setting( 'show_admin' ) ) {
+				return;
+			}
+
+			$page = add_submenu_page( 'edit.php?post_type=acf-field-group', __( 'Experiments', 'secure-custom-fields' ), __( 'Experiments', 'secure-custom-fields' ), acf_get_setting( 'capability' ), 'scf-experiments', array( $this, 'html' ) );
+
+			add_action( 'load-' . $page, array( $this, 'load' ) );
+		}
+
+		/**
+		 * Loads the admin experiments page.
+		 *
+		 * @since   SCF 6.4.2
+		 *
+		 * @return  void
+		 */
+		public function load() {
+			add_filter( 'admin_body_class', array( $this, 'admin_body_class' ) );
+
+			// disable filters (default to raw data)
+			acf_disable_filters();
+
+			// Temporarily disabled - will be enabled when experiments feature is ready
+			// $this->include_experiments();
+
+			$this->check_submit();
+
+			acf_enqueue_scripts();
+
+			// Temporarily disabled - Uncomment to enable in production when ready.
+			// acf_enqueue_script( 'acf-experiments' );
+
+			// Temporarily disabled - Localize experiments data. Uncomment to enable in production when ready.
+			// $this->enqueue_experiments_script();
```

Comment:
What's the relationship between the scripts? Why not to put all of them inside `enqueue_experiments_script` method?

Reply:
`enqueue_experiments_script` localizes the script, passing the experiments from PHP to JS to be read. Let me change it's name to include localize and move the enqueue there.